org.jrebirth.af/core/src/main/java/org/jrebirth/af/core/util/ClasspathUtility.java

```diff
@@ -77,158 +71,17 @@ private ClasspathUtility() {
 
         final List<String> resources = new ArrayList<>();
 
-        final ClassLoader cl = Thread.currentThread().getContextClassLoader();
-
-        if (hasJavaWebstartLibrary() && cl instanceof JNLPClassLoaderIf) {
-
-            LOGGER.log(USE_JNLP_CLASSLOADER);
-
-            final JNLPClassLoaderIf wsLoader = (JNLPClassLoaderIf) cl;
-
-            // System.out.println("URLs "+ wsLoader.getURLs());
-            for (final JARDesc jd : wsLoader.getLaunchDesc().getResources().getLocalJarDescs()) {
-                try {
-
-                    final JarFile jarFile = wsLoader.getJarFile(jd.getLocation());
-
-                    LOGGER.log(PARSE_CACHED_JAR_FILE, jarFile.getName(), jd.getLocationString());
-
-                    resources.addAll(getResources(jarFile.getName(), searchPattern, true));
-                } catch (final IOException e) {
-                    LOGGER.log(CANT_READ_CACHED_JAR_FILE, jd.getLocation());
-                }
-            }
-
-        } else {
-
-            LOGGER.log(USE_DEFAULT_CLASSLOADER);
-
-            final String[] classpathEntries = CLASSPATH.split(CLASSPATH_SEPARATOR);
-            for (final String urlEntry : classpathEntries) {
-                // Parse the classpath entry and apply the given pattern as filter
-                resources.addAll(getResources(urlEntry, searchPattern, false));
-            }
+        try (ScanResult scanResult = new ClassGraph()

```

Comment:
I had to use io.github.classgraph:classgraph here, the previous way to scan for ressources is not working anymore with the new module system

Reply:
This is the first thing I noticed, I don't really like to add a core dependency, but I'm not sure wanting to write again a such piece of code (moreover since JWS ' dead).
But I will double check why the Parameter Test doesn't work.... it could be related... (or not)